### ISSUE TITLE
Installing pai in a python venv (virtual environment)

### ISSUE THREAD

**Issue by gergelyszabo94** [ParadoxAlarmInterface/pai#489]
## Alarm system
Panel: MG5050, access mode IP150+

**Comment by yozik04** [comment]
You do all correctly. You can pass in configuration file location. Run PAI with --help it will tell you possible arguments. I think you will need to pass in -c option with configuration file location.

**Comment by gergelyszabo94** [comment]
@yozik04 You are right, I should be able to set the configuration file location using the -c option, butI can't find my configuration file.
I did:
```python3 -m venv --system-site-packages paivenv
source paivenv/bin/activate
pip3 install paradox-alarm-interface
```
inside the homebridge directory. So after activating the venv it looks like this:
```(paivenv) pi@homebridge:/var/lib/homebridge/paivenv $```
I can't find the config.

**Comment by yozik04** [comment]
Copy from default one.
and follow the doc https://github.com/ParadoxAlarmInterface/pai/wiki/Configuration
